rapid7/metasploit-framework
# MS9177 MSSQL part 1 - client consolidation

In the name of making MSSQL client more consistent to other client types, this moves modules that rely on `Remote::MSSQL` to the new consolidated client.

To test:
- Load up any/each module changed in this pr, ie `mssql_login`
- Set up an MSSQL instance in a VM/Target
- set `RHOSTS` to the ip of the MSSQL target
- set `RPORT` to 1433, `USERNAME` and `PASSWORD` to MSSQL credentials
- run the module, ensure no difference between here and master
